<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow</class>
 <widget class="QMainWindow" name="mainwindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Inventory++</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTableView" name="t_progs">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="alternatingRowColors">
         <bool>false</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="horizontalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>25</number>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="m_exit"/>
   </widget>
   <widget class="QMenu" name="m_edit">
    <property name="title">
     <string>Правка</string>
    </property>
    <widget class="QMenu" name="m_subprogs">
     <property name="title">
      <string>Программы</string>
     </property>
     <addaction name="m_gprogs"/>
     <addaction name="separator"/>
     <addaction name="m_wlist"/>
     <addaction name="m_blist"/>
     <addaction name="separator"/>
     <addaction name="m_hidden"/>
    </widget>
    <addaction name="m_gcomps"/>
    <addaction name="separator"/>
    <addaction name="m_subprogs"/>
   </widget>
   <widget class="QMenu" name="m_info">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="m_help"/>
    <addaction name="separator"/>
    <addaction name="action_11"/>
   </widget>
   <widget class="QMenu" name="m_view">
    <property name="title">
     <string>Вид</string>
    </property>
    <addaction name="m_colors"/>
    <addaction name="m_showhidden"/>
   </widget>
   <widget class="QMenu" name="m_reports">
    <property name="title">
     <string>Отчёты</string>
    </property>
    <addaction name="m_qreport"/>
    <addaction name="m_browser"/>
   </widget>
   <widget class="QMenu" name="m_settings">
    <property name="title">
     <string>Настройки</string>
    </property>
    <addaction name="m_users"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="m_view"/>
   <addaction name="m_edit"/>
   <addaction name="m_reports"/>
   <addaction name="m_settings"/>
   <addaction name="m_info"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget_5">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>69</width>
     <height>47</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>160</width>
     <height>10182</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>160</width>
      <height>16777215</height>
     </size>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_4">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>166</width>
     <height>187</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>180</width>
     <height>995</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>180</width>
      <height>9999</height>
     </size>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QComboBox" name="cb_gcomps">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Группы компьютеров</string>
       </property>
       <property name="whatsThis">
        <string>В этом списке перечислены группы компьютеров</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="t_comps">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>4</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>9999</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Список компьютеров</string>
       </property>
       <property name="whatsThis">
        <string>Это список компьютеров, находящихся в выбранной группе</string>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="m_exit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/buttons/exit.png</normaloff>:/images/buttons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="statusTip">
    <string>Выйти из программы</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="m_gcomps">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/buttons/gcomps.png</normaloff>:/images/buttons/gcomps.png</iconset>
   </property>
   <property name="text">
    <string>Группы ПК</string>
   </property>
   <property name="statusTip">
    <string>Редактировать группы компьютеров</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Группы</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>Белый список</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>Чёрный список</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>Справка</string>
   </property>
  </action>
  <action name="action_11">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/buttons/about.png</normaloff>:/images/buttons/about.png</iconset>
   </property>
   <property name="text">
    <string>О программе</string>
   </property>
   <property name="statusTip">
    <string>Открыть окно &quot;О программе&quot;</string>
   </property>
   <property name="whatsThis">
    <string>В данном окне вы сможете узнать более подробно о программе и её авторе</string>
   </property>
  </action>
  <action name="m_gprogs">
   <property name="text">
    <string>Группы программ</string>
   </property>
   <property name="statusTip">
    <string>Редактировать группы программ</string>
   </property>
  </action>
  <action name="m_wlist">
   <property name="text">
    <string>Белый список</string>
   </property>
  </action>
  <action name="m_blist">
   <property name="text">
    <string>Чёрный список</string>
   </property>
  </action>
  <action name="m_colors">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Цветовая схема</string>
   </property>
   <property name="toolTip">
    <string>Цветовая схема</string>
   </property>
   <property name="statusTip">
    <string>Включить/отключить цветовую схему для основной таблицы</string>
   </property>
   <property name="whatsThis">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Bitstream Vera Sans'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Данный чекбокс включает/отключает цветовую схему основной таблицы.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;При включенной цветовой схеме:&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#fff4d7; background-color: #5e5e5e;&quot;&gt;Таким цветом помечаются запрещённые программы.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#d0ffd0; background-color: #5e5e5e;&quot;&gt;Таким цветом помечаются разрешённые программы&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#d8d8d8; background-color: #5e5e5e;&quot;&gt;Таким цветом помечаются нейтральные программы&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="m_qreport">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/buttons/printer.png</normaloff>:/images/buttons/printer.png</iconset>
   </property>
   <property name="text">
    <string>Быстрый отчёт</string>
   </property>
   <property name="statusTip">
    <string>Создать отчёт по выбранному компьютеру</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="m_browser">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/buttons/reportbrowser.png</normaloff>:/images/buttons/reportbrowser.png</iconset>
   </property>
   <property name="text">
    <string>Просмотрщик отчётов</string>
   </property>
   <property name="statusTip">
    <string>Просмотреть отчёты, сохранённые на компьютере</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="m_hidden">
   <property name="text">
    <string>Список скрытых</string>
   </property>
   <property name="statusTip">
    <string>Редактировать список скрытых программ</string>
   </property>
  </action>
  <action name="m_showhidden">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Показывать скрытые</string>
   </property>
   <property name="statusTip">
    <string>Отображать/спрятать &quot;скрытые&quot; программы</string>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
  </action>
  <action name="m_users">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/buttons/user-password.png</normaloff>:/images/buttons/user-password.png</iconset>
   </property>
   <property name="text">
    <string>Пользователи</string>
   </property>
   <property name="toolTip">
    <string>Редактировать пароли пользователей</string>
   </property>
   <property name="statusTip">
    <string>Редактировать пароли пользователей</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="m_help">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/buttons/help-contents.png</normaloff>:/images/buttons/help-contents.png</iconset>
   </property>
   <property name="text">
    <string>Содержание</string>
   </property>
   <property name="statusTip">
    <string>Вызвать окно справки</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>t_progs</tabstop>
  <tabstop>cb_gcomps</tabstop>
  <tabstop>t_comps</tabstop>
 </tabstops>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
